Project.toml

```diff
@@ -14,7 +14,7 @@ Flux = "0.10.4, 0.11"
 Lighthouse = "0.11"
 StableRNGs = "1.0"
 Zygote = "0.4.13, 0.5"
-julia = "1.3"
+julia = "1.3, 1.6"
```

Comment:
This change isn't necessary. When you list a version in Project.toml with no leading qualifier like `=` or `~`, you're saying that you're allowing any version that is semver-compatible with what's listed. Since 1.(x + y) is semver compatible with 1.x, just saying 1.3 in this case is sufficient to cover 1.6.